index.html

```diff
@@ -0,0 +1,366 @@
+          <span class="panel-span">A</span>
+
+          <p class="answer">
+            SKOUT will work on Windows, Mac, and Linux environments.
+          </p>
+        </div>
+      </div>
+      <div>
+        <div class="head-accordion-Faqs">
+          <span>Q</span>
+          <span class="question">Is there a trial program?</span>
+          <button class="accordion-button"></button>
+        </div>
+        <div class="panel">
+          <span class="panel-span">A</span>
+
+          <p class="answer">
+            SKOUT will work on Windows, Mac, and Linux environments.
+          </p>
+        </div>
+      </div>
+      <div>
+        <div class="head-accordion-Faqs">
+          <span>Q</span>
+          <span class="question">Can I get a Demo?</span>
+          <button class="accordion-button"></button>
+        </div>
+        <div class="panel">
+          <span class="panel-span">A</span>
+
+          <p class="answer">
+            SKOUT will work on Windows, Mac, and Linux environments.
+          </p>
+        </div>
+      </div>
+      <div class="more-questions">MORE QUESTIONS</div>
+    </div>
+
+    <footer>
+      <div class="footer-wrapper">
+        <div class="about-company">
+          <ul class="link-company">
+            <li><a href="#">Blog</a></li>
+            <li><a href="#">Company</a></li>
+            <li><a href="#">Terms & Conditions</a></li>
+            <li><a href="#">Privacy Policy</a></li>
+            <li><a href="#">Careers</a></li>
+            <li>
+              <a href="tel:1-855-838-4500" id="Call"
+                >Call the SOC<br /><span>1-855-838-4500</span></a
+              >
+            </li>
+          </ul>
+          <ul class="address-container">
+            <li class="address">
+              <p>New York</p>
+              <a href="tel:+1 631-203-6600" class="phone">+1 631-203-6600</a>
+              <div>
+                <span>423 W 55th St. 7th Floor</span>
+                <span>New York, NY, 10019</span>
+              </div>
+              <a href="#" class="view-map">View Map</a>
+            </li>
+            <li class="address">
+              <p>New York</p>
+              <a href="tel:+1 631-203-6600" class="phone">+1 631-203-6600</a>
+              <div>
+                <span>423 W 55th St. 7th Floor</span>
+                <span>New York, NY, 10019</span>
+              </div>
+              <a href="#" class="view-map">View Map</a>
+            </li>
+            <li class="address">
+              <p>New York</p>
+              <a href="tel:+1 631-203-6600" class="phone">+1 631-203-6600</a>
+              <div>
+                <span>423 W 55th St. 7th Floor</span>
+                <span>New York, NY, 10019</span>
+              </div>
+              <a href="#" class="view-map">View Map</a>
+            </li>
+          </ul>
+        </div>
+        <div class="social">
+          <ul class="conteiner-social-icons">
+            <li class="twitter"></li>
```

Comment:
inside must be a link to the following sites

Reply:
added link